data/campaigns/Winds_of_Fate/scenarios/03_The_Contention.cfg

```diff
@@ -0,0 +1,471 @@
+                        remove_empty=yes
+                    [/join]
+                [/set_variable]
+                [allow_recruit]
+                    side=$unit.side
+                    type=$recruit
+                [/allow_recruit]
+                {CLEAR_VARIABLE (recruit,$unit.side|_recruit)}
+            [/then]
+        [/if]
+        {CLEAR_VARIABLE side}
+    [/event]
+
+    # Leave the leaders on the map for final dialog
+    [event]
+        name=last breath
+        [filter]
+            id=Gorlack
+        [/filter]
+        {MODIFY_UNIT id=$unit.id hitpoints 1}
+        [endlevel]
+            result=defeat
+        [/endlevel]
+    [/event]
+
+    [event]
+        name=last breath
+        [filter]
+            id=Karron
+        [/filter]
+        {MODIFY_UNIT id=$unit.id hitpoints 1}
+        {VARIABLE leader_kill yes}
+        [endlevel]
+            result=victory
+        [/endlevel]
+    [/event]
+
+    # This event is last breath to prevent death animation
+    # Place units on the recall list if they are defeated.
+    [event]
+        name=last breath
+        first_time_only=no
+        [filter]
+            [not]
+                id=Gorlack,Karron
+            [/not]
+        [/filter]
+        [put_to_recall_list]
+            id=$unit.id
+            heal=yes
+        [/put_to_recall_list]
+    [/event]
+
+    [event]
+        name=time over
+        first_time_only=no
+        # Determine which side has the advantage
+        [lua]
+            code = << wml.variables["result"] = wesnoth.dofile("campaigns/Winds_of_Fate/lua/lua.lua") >>
+        [/lua]
+        [switch]
+            variable=result
+            [case]
+                value=1
+                [endlevel]
+                    result=victory
+                [/endlevel]
+            [/case]
+            [case]
+                value=2
+                [endlevel]
+                    result=defeat
+                [/endlevel]
+            [/case]
+            [case]
+                value=tie
+                [modify_turns]
+                    add=2
+                [/modify_turns]
+            [/case]
+            [else]
+                # error, do nothing and end in defeat
+            [/else]
+        [/switch]
+        {CLEAR_VARIABLE result}
+    [/event]
+
+    [event]
+        name=defeat
+        [message]
+            speaker=Karron
+            image_pos=right
+            mirror=yes
+            message= _ "This must be a dream. I no longer dwell in the shadow of Gorlack."
+        [/message]
+    [/event]
+
+    [event]
+        name=victory
+        [message]
+            speaker=Karron
+            image_pos=right
+            mirror=yes
+            message= _ "........."
+        [/message]
+        [message]
+            speaker=Gorlack
+            message= _ "Drakes!
+
+Fellows of all castes, hear me!
+
+This is a dark time in which we endure.
+Our lands provide not as once they did.
+Unfed hatchlings succumb to the wasting.
```

Comment:
The only other instance of wasting that I could find is at the very start of the campaign where it is capitalised. It's never explained (that I could see) what the Wasting is supposed to be, but it should probably be capitalised here if it is capitalised in the introduction.

Reply:
Given that "unfed" is in the sentence, I'd read this is "starvation".

Reply:
Sure, but my issue is with the inconsistent capitalisation, which I'll also bring up later in my general remarks.

Reply:
made "wasting" consistently lowercase in both instances